docs/concepts/terminology.md

```diff
@@ -0,0 +1,215 @@
+composite resource, but without the "Composite" prefix. So an "AcmeCoDB" would
+be a type of claim, and a "CompositeAcmeCoDB" would be the corresponding type of
+composite resource. This allows claim consumers to be relatively ignorant of
+Crossplane and composition, and to instead simply think about managing “an
+AcmeCo DB” while the platform team worries about the implementation details.
+
+The term “Composite Resource Claim” refers to a class of types, so avoid using
+Pascal case - “Composite Resource Claim” not CompositeResourceClaim. Use Pascal
+case when referring to a distinct type of composite resource claim - e.g. an
+AcmeCoDB.
+
+> Claims map to the same concepts as described above under the composite
+> resource heading; i.e. `tfvars` files and Helm `values.yaml` files. Imagine
+> that some `tfvars` files and some `values.yaml` files were only accessible to
+> the platform team while others were offered to application teams; that’s the
+> difference between a composite resource and a claim.
+
+### Composite Resource Definition
+
+A “Composite Resource Definition” or “XRD” is the API type used to define new
+types of composite resources and claims. Types of composite resources and types
+of claims exist because they were defined into existence by an XRD. The XRD
+configures Crossplane with support for the composite resources and claims that
+make up a platform API.
+
+XRDs are often conflated with composite resources (XRs) - try to avoid this.
+When someone uses the platform API to create infrastructure they’re not creating
+XRDs but rather creating composite resources (XRs). It may help to think of a
+composite resource as a database entry, while an XRD is a database schema. For
+those familiar with Kubernetes, the relationship is very similar to that between
+a Custom Resource Definition (CRD) and a Custom Resource (CR).
+
+A CompositeResourceDefinition is a distinct Crossplane API type, so Pascal case
+and fixed width fonts are appropriate.
+
+> There isn’t a direct analog to XRDs in the Helm ecosystem, but they’re a
+> little bit like the variable blocks in a Terraform module that define which
+> variables exist, whether those variables are strings or integers, whether
+> they’re required or optional, etc.
+
+### Managed Resource
+
+Managed resources are granular, high fidelity Crossplane representations of a
+resource in an external system - i.e. resources that are managed by Crossplane.
+Managed resources are what Crossplane enables platform teams to compose into
+higher level composite resources, forming an opinionated platform API. They're
+the building blocks of Crossplane.
+
+You’ll often hear three related terms used in the Crossplane ecosystem; composed
+resource, managed resource, and external resource. While there are some subtle
+contextual differences, these all broadly refer to the same thing. Take an
+RDSInstance for example; it is a managed resource. A distinct resource within
```

Comment:
nitpick:
```suggestion
RDSInstance for example; it is a managed resource - a distinct resource within
```

Reply:
I think I avoided this because mixing the earlier semicolon and the hypen felt... odd.

Reply:
ha fair point :) I was just a little tripped up with what I think is an incomplete sentence